Description: Increase timeout for "ejabberdctl dump"
 Increases the timeout of the "ejabberdctl dump" command, which now frequently
 times out because of the large new mod_mam tables.
Author: Mathias Ertl <mati@jabber.at>
Bug: https://github.com/processone/ejabberd/issues/866
Last-Update: 2015-12-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/ejabberd_ctl.erl
+++ b/src/ejabberd_ctl.erl
@@ -80,7 +80,7 @@
 			     end
 		     end,
 	    Node = list_to_atom(SNode1),
-	    Status = case rpc:call(Node, ?MODULE, process, [Args], 60000) of
+	    Status = case rpc:call(Node, ?MODULE, process, [Args], 600000) of
 			 {badrpc, Reason} ->
 			     print("Failed RPC connection to the node ~p: ~p~n",
 				    [Node, Reason]),
